pkg/rpcclient/wsclient_test.go

```diff
@@ -68,6 +69,9 @@ func TestWSClientSubscription(t *testing.T) {
 		"notary requests": func(wsc *WSClient) (string, error) {
 			return wsc.ReceiveNotaryRequests(nil, ntrCh)
 		},
+		"mempool": func(wsc *WSClient) (string, error) {
+			return wsc.ReceiveMempool(nil, mempoolCh)
+		},
```

Comment:
Good, this testcase is extended in a valid way. No more modifications are required because here we completely mock RPC server behaviour.